ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -183,6 +183,17 @@ def install_mcg_only_cluster(self):
         self.do_click(self.dep_loc["next"], enable_screenshot=True)
         self.create_storage_cluster()
 
+    def configure_in_transit_encryption(self):
+        """
+        Configure in_transit_encryption
+
+        """
+        if config.ENV_DATA.get("in_transit_encryption"):
+            logger.info("Enable in-transit encryption")
+            self.select_checkbox_status(
+                status=True, locator=self.dep_loc["enable_in_transit_encryption"]
```

Comment:
@OdedViner  
Do we need to disable it in the end of the test / tearDown? 
Will it affect other test cases running on cluster? 
Is the feature "in-transit encryption" applicable for all deployments?

Reply:
@DanielOsypenko For UI this option ( to enable in-transit encryption) is available only during deployment not after that,  so we cant disable it form UI.  It should not affect the other tests cases , If it is then those are real issue we need to investigate.

Reply:
ack, thanks